### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by yveslaroche**
I updated the following in the templates folder:
- index.html, 404.html and htaccess to reflect the changes made to the H5BP [4.2.0](https://github.com/h5bp/html5-boilerplate/blob/master/CHANGELOG.md#420-april-8-2013)
- bootstrap.js to 2.3.1
- bower components
- npm devDependencies

### CODE REVIEW

File: app/templates/_component.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "name": "<%= _.slugify(appname) %>",
   "version": "0.0.0",
   "dependencies": {<% if (compassBootstrap) { %>
-    "sass-bootstrap": "~2.3.0",<% } %><% if (includeRequireJS) { %>
-    "requirejs": "~2.1.4",<% } %>
+    "sass-bootstrap": "~2.3.1",<% } %><% if (includeRequireJS) { %>
+    "requirejs": "~2.1.5",<% } %>
```
Comment (passy) [inline app/templates/_component.json]: You don't actually need to update these for dot-releases, they will get picked up automatically if they're prefixed with `~`.

File: app/templates/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
         <script>
             var _gaq=[['_setAccount','UA-XXXXX-X'],['_trackPageview']];
             (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
-            g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
+            g.src='//www.google-analytics.com/ga.js';
```
Comment (passy) [inline app/templates/index.html:27]: Awesome! Didn't know they finally fixed this.

File: app/templates/_component.json (hunk 1 above)
Comment (kevva) [inline app/templates/_component.json]: Shouldn't we use the Bootstrap generator anyway?

File: app/templates/_package.json [hunk 3]
```diff
@@ -5,28 +5,28 @@
...
+    "grunt-mocha": "~0.3.1",<% } %>
     "grunt-open": "~0.2.0",
     "grunt-svgmin": "~0.1.0",
-    "grunt-concurrent": "~0.1.0",
-    "matchdep": "~0.1.1"
+    "grunt-concurrent": "~0.1.1",
+    "matchdep": "~0.1.2"
   },
   "engines": {
     "node": ">=0.8.0"
```
Comment (sindresorhus) [inline app/templates/_package.json:32]: Thanks, but we update dependencies on releases.

Comment (yveslaroche) [inline app/templates/_package.json:32]: Ok, sorry I didn't know this. I've removed the commit.

File: app/templates/_component.json (hunk 1 above)
Comment (passy) [inline app/templates/_component.json]: Eventually, yes. But there's still a bit of work to do.

Comment (yveslaroche) [inline app/templates/_component.json]: @passy Ok, thanks for the info. I removed the specific commit.

**Comment by yveslaroche** [comment]
Removed the devDependencies and bower commits. This is actually the first pull-request I have made with git, so hope everything is fine now :)

**Comment by sindresorhus** [comment]
Thanks man. And congrats on your first PR. Keep rolling :)

:cake:

**Comment by passy** [comment]
Congrats, @yveslaroche! :)

:lollipop:

**Pull request closed without merging**